test/cachingTest.js

```diff
@@ -70,12 +70,12 @@ describe( 'iinChecker()', function() {
             throw ( err );
           } else {
             expect( result ).to.deep.equal( cardDetails );
-            expect( setCacheStub.calledOnce ).to.be.true;
+            expect( setCacheStub.calledOnce ).to.equal(true);
```

Comment:
If you use dirty-chai you can keep these as 
```
expect( setCacheStub.calledOnce ).to.be.true();
```

We have used this quite a bit in other repos